pytest_datafiles.py

```diff
@@ -5,6 +5,11 @@
 from py import path  # pylint: disable=E0611
 import pytest
 
+def pytest_configure(config):
+    config.addinivalue_line("datafile", "datafile: use tmp data files.")
```

Comment:
The build probably fails because it should be "datafiles" (plural).

Reply:
Thanks, done.